virttest/utils_windows/drive.py

```diff
@@ -23,7 +25,7 @@ def get_hard_drive_letter(session, label):
     cond = "VolumeName like '%s'" % label
     try:
         return _logical_disks(session, cond=cond, props=["DeviceID"])[0]
-    except IndexError:
+    except TypeError:
```

Comment:
With the above change, we don't need this anymore.

Reply:
Updated, thanks~